README.md

```diff
@@ -62,10 +88,16 @@ Salary slip kata
            Taxable income: £2,833.33
            Tax Payable: £600.00
 
-#### Scenario 5: ...for an employee earning an annual salary of £101,000.00    
-  ...
-  <p>Then the monthly salary slip should contain the below:</p>
-   
+Calculation rules:
+ * Taxable income (higher rate): Any amount of money earned above a gross annual salary of £43,000.00 is taxed at 40% instead of 20%
```

Comment:
This might be incorrectly interpreted, so don't say `instead of ..%` because salaries above £43,000.00 are applicable for both bands of tax deductions, you pay both 20% and 40% of the taxes for portions of your salary.